.yamllint

```diff
@@ -12,6 +12,8 @@ ignore: |
   node_modules/
   test/**/states/**/*.sls
   .kitchen/
+  kitchen.yml
```

Comment:
This can be be avoided using the proposal mentioned below.

Reply:
Done.